### PR TITLE
notifications about new versions of plugins

### PR DESCRIPTION
**Pull request by Hidanio**

### CODE REVIEW

File: src/main/kotlin/com/vk/admstorm/AdmStormStartupActivity.kt [hunk 1]
```diff
@@ -67,26 +68,36 @@ class AdmStormStartupActivity : ProjectActivity {
         Logger.setFactory(AdmStormLoggerFactory(sentry, defaultLoggerFactory))
     }
 
+    private fun hasNewVersion(pluginId : PluginId): Boolean {
```
Comment (Danil42Russia) [inline src/main/kotlin/com/vk/admstorm/AdmStormStartupActivity.kt:71]: Let's put the check for new versions in a separate class?

**Comment by Danil42Russia** [comment]
@Hidanio, you have a branch conflict

File: src/main/kotlin/com/vk/admstorm/services/SentryService.kt [hunk 2]
```diff
@@ -34,7 +34,7 @@ class SentryService(project: Project) {
 
     init {
```
Comment (Danil42Russia) [inline src/main/kotlin/com/vk/admstorm/services/SentryService.kt:35]: Conflict in the file

File: src/main/kotlin/com/vk/admstorm/AdmStormStartupActivity.kt (hunk 1 above)
Comment (Hidanio) [reply src/main/kotlin/com/vk/admstorm/AdmStormStartupActivity.kt]: Done

File: src/main/kotlin/com/vk/admstorm/services/SentryService.kt (hunk 2 above)
Comment (Hidanio) [reply src/main/kotlin/com/vk/admstorm/services/SentryService.kt]: Resolved

**Review by Danil42Russia — APPROVED** [review]

**Pull request merged**